lib/internal/test_runner/tap_stream.js

```diff
@@ -105,11 +126,19 @@ class TapStream extends Readable {
       line += ` ${tapEscape(`- ${name}`)}`;
     }
 
-    line += ArrayPrototypeJoin(ArrayPrototypeMap(ObjectEntries(directive), ({ 0: key, 1: value }) => (
-      ` # ${key.toUpperCase()}${value ? ` ${tapEscape(value)}` : ''}`
-    )), '');
+    line += ArrayPrototypeJoin(
```

Comment:
many unrelated changes, can you please reduce the diff?

Reply:
done.